### DISCUSSION TITLE
Spectral Library using categorized point shapefiles

### DISCUSSION THREAD

**Discussion by dpoursanidis** [EnMAP-Box/enmap-box#181] [General]
Hi,

I have a multiband imagery and a set of categorized (level1, level2) point shapefile.
Is possible to make "automatically" using those two as inputs, a spectar library and interactively to remove the points that have bizzare spectral curves?

Best
Dimitris

**Comment by janzandr** [comment]
Not quite sure what you want to achieve. Please show a step by step description using screenshot or an animation.

**Comment by thielfab** [comment]
If I understand correctly, you can do this like this:
![grafik](https://user-images.githubusercontent.com/15610561/195277233-9a80c919-f946-4bae-ad29-426a1b2a9514.png)
Confirm with OK and a spectral library will open with the profiles extracted at the point location + corresponding attribute values. You may then add the library as vector geometries again on the map view by right-clicking into a map view --> `Add Spectral Library`. This also allows you to use `Zoom to Feature` and other functionalities in the context menu of the spectral library view attribute table.

**Comment by dpoursanidis** [comment]
Yeap! Was so easy but not straightforward to find it!

Maybe it worth to make a full manual for EnMAP!!!

**Reply by thielfab** [reply]
Yes, you are right. Documentation is crucial, and we are working on improving it :-)
